interfaces/portal/src/app/pages/project-registrations/project-registrations.page.ts

```diff
@@ -169,9 +183,25 @@ export class ProjectRegistrationsPageComponent {
     });
 
     if (!actionData) {
+      this.trackingService.trackEvent({
+        category: TrackingCategory.manageRegistrations,
+        action: triggeredFromContextMenu
+          ? TrackingAction.selectContextMenuOption
+          : TrackingAction.clickBulkActionButton,
+        name: `send-message for:none`,
```

Comment:
can you not just combine this in the same trackEvent scenario where the count would be 0? Or if not, make the name a bit more descriptive, namely that the 'action is declined because of no selection' or something? (Specifically, if I interpret the refinement/description in the main item correctly this is the top-prioritized event to track.)

Reply:
Well... it comes down to "what information is available at what moment"..

At the moment(in the code) where the Toast is thrown, the information whether it is triggered via the context-menu OR the "specific action a user wanted" was not (easily) available.

I think therefore that the coded ended up like it did here...

Also in an attempt to get the contents of the `name`-properties a bit aligned / 'parse-able' across all the different ways to trigger a similar Action...
But it is in no way perfect. :D

Reply:
ok, sure :)